x-pack/legacy/plugins/fleet/public/pages/agent_list/index.tsx

```diff
@@ -181,6 +187,8 @@ export const AgentListPage: React.SFC<{}> = () => {
         defaultMessage: 'Policy',
       }),
       truncateText: true,
+      render: (policyId: string) =>
+        (policies.find(p => p.id === policyId) || ({} as Policy)).name || `Policy: ${policyId}`,
```

Comment:
I think just `|| policyId` will suffice, but if we want to prefix it with `Policy:` then we should localize the string

Reply:
> Do you think it makes sense to return this kind of information in the policy GET endpoint? 
I would rather keep these concerns separate... GraphQL would be nice here to pull both in one request, but for now we will need to pull from multiple endpoints.

> Total agents enrolled into the policy could be retrieved by querying the agent list endpoint, but I'm not sure if # of events 
The events API can return this

>and status summary across all agents enrolled in a particular policy can be retrieved from the agent list query.
This is a good question... I am starting to thing the agent listing and agent get API both return status for each agent vs the calculations in the UI. Also, maybe the aggregated status?

Reply:
> I am starting to thing the agent listing and agent get API both return status for each agent vs the calculations in the UI. Also, maybe the aggregated status?

This information makes sense to me to return in APIs (both status per agent + aggregated status). What do you think @nchaulet?

Reply:
I am totally for build agent status server side 👍

For the stats, I think it will make sense to have a dedicated API for that something like
```
/api/fleet/agents-status?policy_id={policyId}
{
"statuses": {
  "running": 12,
  "error": 12,
},
"total": 10
}
```

Reply:
I like that plan

Reply:
I am going to create issues for that